### PR TITLE
UF-9748: Added mapping of 25 years employment

### PR DESCRIPTION
**Pull request by MickeWallner**

### CODE REVIEW

File: src/main/resources/application-it.yml [hunk 1]
```diff
@@ -59,7 +59,13 @@ kontakt-lon-pension:
   family-id: 121
   priority: MEDIUM
   category: SALARY
-  type: PENSION
+  type: OTHER
+tjugofem-ar-pa-jobbet:
```
Comment (foobar-1337) [inline src/main/resources/application-it.yml:63]: Swedish? The java code defines this as "twentyfiveatwork".
Maybe stick to the same language here?

File: src/main/java/se/sundsvall/smloader/integration/openemapper/OpenEMapperConfiguration.java [hunk 2]
```diff
@@ -42,4 +42,10 @@ public OpenEMapperProperties propertiesSalaryChange() {
 	public OpenEMapperProperties propertiesContactSalaryAndPension() {
 		return new OpenEMapperProperties();
 	}
+
+	@Bean(name = "twentyfiveatwork")
+	@ConfigurationProperties(prefix = "tjugofem-ar-pa-jobbet")
+	public OpenEMapperProperties propertiesTwentyFiveAtWork() {
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/smloader/integration/openemapper/OpenEMapperConfiguration.java:48]: `public`-keyword not necessary for `@Bean`-methods, right?

File: src/test/resources/open-e/flow-instance-25-pa-jobbet-xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
```
Comment (foobar-1337) [inline src/test/resources/open-e/flow-instance-25-pa-jobbet-xml:1]: Indent?

File: src/main/resources/application-it.yml (hunk 1 above)
Comment (MickeWallner) [reply src/main/resources/application-it.yml:63]: For being consistent with the other services and easy identify corresponding OpenE-service.

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**